### ISSUE TITLE
Merge issues with Botw - Intel Flipped lights, RemoveBloom

### ISSUE THREAD

**Issue by getdls** [slashiee/cemu_graphic_packs#348]
I'm running into merge issues with Botw intel fixes even after  a 
```
git reset --hard HEAD
git fetch --all


```
I assume its related to most of them missing new line , any suggestions on how to proceed?  I've tried the nuke all local changes approach,  but after a sync I'm back to  all of the intel fixes shown as  (new) changes. 

```
}
\ No newline at end of file
```

**Comment by slashiee** [comment]
On mobile so @Alstura

**Comment by skalfate** [comment]
Oh, Is that why i was getting the same issues on my github desktop application, so its the new line at the end huh not being added.

**Comment by skalfate** [comment]
I have let him know, he'll respond sometime soon.

**Comment by Alstura** [comment]
@getdls feel free to add the new lines and merge

**Comment by Crementif** [comment]
Well, I tried fixing those recently (see https://github.com/slashiee/cemu_graphic_packs/commit/e0fa6b5a3bb519d34a29f0a3ac07be039fa7753f) and I'm actually not getting them anymore. Maybe try deleting your local repository and then cloning them. Also, if you're using Linux (which afaik you are not) it could be an issue with <https://github.com/slashiee/cemu_graphic_packs/blob/master/.gitattributes> which forces the line endings? Other then that, I've got no clue. It's pretty annoying to have these ghost files.

Just shouldn't use the web editor from now on, I guess 👍
